purchases_ui_flutter/lib/paywall_result.dart

```diff
@@ -0,0 +1,18 @@
+/// Possible values for the result of a paywall presentation.
+enum PaywallResult {
+  /// Only returned when using [RevenueCatUI.presentPaywallIfNeeded]. Returned
+  /// if the paywall was not presented.
+  notPresented,
+  /// Returned when the paywall was presented and the user cancelled without
+  /// executing an action.
+  cancelled,
```

Comment:
What if you tap on purchase, cancel, and then purchase again? Would you only get this?

Reply:
You would only get the last result before the paywall closed. In this case, the purchase.

Reply:
Ooooh when it's closed, got it 👍🏻